### PR TITLE
271 accessibility

### LINKED ISSUE #271: Check accessibility of the lesson
- [x] The alternative text of all figures is accurate and sufficiently detailed *.
  - Large and/or complex figures may not be described completely in the alt text of the image and instead be described elsewhere in the main body of the episode.
- [x] The lesson content does not make extensive use of colloquialisms, region- or culture-specific references, or idioms.
- [x] The lesson content does not make extensive use of contractions (“can’t” instead of “cannot”, “we’ve” instead of “we have”, etc).

\* To view the alternative text of an image, we recommend using
[the WAVE Web Accessibility Evaluation Tool][wave] or associated browser extensions.
You can also _inspect_ the source HTML of the image element in the developer tools of your web browser,
or consult the source (R)Markdown file for the relevant page in the lesson repository on GitHub.
For more information about what makes good alternative text for an image,
read [How to Design Great Alt Text: An Introduction][deque-alt-text],
and [Writing Alt Text for Data Visualization][alt-text-data]

[alt-text-data]: https://medium.com/nightingale/writing-alt-text-for-data-visualization-2a218ef43f81
[code-of-conduct]: https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html
[deque-alt-text]: https://www.deque.com/blog/great-alt-text-introduction/
[glosario]: https://carpentries.github.io/glosario/
[wave]: https://wave.webaim.org/

### PR DESCRIPTION
**Pull request by cpranav93**
Fixes #271 
Modified all alt-text for images making it more explicit and well-defined.

Removed contractions leaving only a few as meaningful.

### CODE REVIEW

**Comment by cpranav93** [comment]
Could someone who knows the objectives, resolve conflict in ep1?

File: _episodes/03-monitor-the-model.md [hunk 1]
```diff
@@ -454,7 +454,7 @@ plot_predictions(y_baseline_prediction, y_test, title='Baseline predictions on t
 ~~~
 {: .language-python}
 
-![Output of plotting sample](../fig/03_regression_test_5_naive_baseline.png){: width="500px"}
+![Scatter plot of predicted vs true sunshine hours in Basel for the test set where the today's sunshine hours is considered as the true sunshine hours for tomorrow](../fig/03_regression_test_5_naive_baseline.png){: width="500px"}
```
Comment (svenvanderburg) [inline _episodes/03-monitor-the-model.md:457]:
```suggestion
![Scatter plot of predicted vs true sunshine hours in Basel for the test set where today's sunshine hours is considered as the true sunshine hours for tomorrow](../fig/03_regression_test_5_naive_baseline.png){: width="500px"}
```

File: _episodes/04-advanced-layer-types.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ plt.show()
 {: .language-python}
 
 
-![Output of plotting sample](../fig/04_cifar10.png)
+![A 5 by 5 grid of 25 sample images from the CIFAR-10 data-set](../fig/04_cifar10.png)
```
Comment (svenvanderburg) [inline _episodes/04-advanced-layer-types.md:156]:
```suggestion
![A 5 by 5 grid of 25 sample images from the CIFAR-10 data-set. Each image is labelled with a category, for exmaple: 'frog' or 'horse'.](../fig/04_cifar10.png)
```

File: _episodes/04-advanced-layer-types.md [hunk 3]
```diff
@@ -496,14 +496,14 @@ Note that the training loss continues to decrease, while the validation loss sta
 Techniques to avoid overfitting, or to improve model generalization, are termed **regularization techniques**.
 One of the most versatile regularization technique is **dropout** ([Srivastava et al., 2014](https://jmlr.org/papers/v15/srivastava14a.html)).
 Dropout essentially means that during each training cycle a random fraction of the dense layer nodes are turned off. This is described with the dropout rate between 0 and 1 which determines the fraction of nodes to silence at a time.
-![Dropout sketch](../fig/neural_network_sketch_dropout.png)
+![A sketch of NN with and without dropout](../fig/neural_network_sketch_dropout.png)
```
Comment (svenvanderburg) [inline _episodes/04-advanced-layer-types.md:499]:
```suggestion
![A sketch of a neural network with and without dropout](../fig/neural_network_sketch_dropout.png)
```

**Review by svenvanderburg — APPROVED** [review]
Wow, this is great! Thanks @cpranav93 :) I added a few small suggestions.

**Pull request merged**